feature_engine/timeseries/forecasting/window_features.py

```diff
@@ -121,6 +121,28 @@ class WindowFeatures(BaseForecastTransformer):
     pandas.rolling
     pandas.aggregate
     pandas.shift
+
+    Examples
+    --------
+
+    >>> import pandas as pd
+    >>> from feature_engine.timeseries.forecasting import WindowFeatures
+    >>> X = pd.DataFrame(dict(date = ["2022-09-18",
+    >>>                             "2022-10-27",
+    >>>                             "2022-12-24",
+    >>>                             "2023-01-09",
+    >>>                             "2023-02-12"],
+    >>>                     x1 = [1,2,3,4,5],
+    >>>                     x2 = [6,7,8,9,10]
+    >>>                     ))
+    >>> wf = WindowFeatures(window = 2)
+    >>> lf.fit_transform(X)
+              date  x1  x2  x1_lag_1  x2_lag_1  x1_lag_2  x2_lag_2
+    0  2022-09-18   1   6       NaN       NaN       NaN       NaN
```

Comment:
I think the output is wrong. It should say window or something different in the name of the new features. Also I think the values might be wrong. Could you double check?

Also, could you make the timestamps consecutive in the dataframe?